dhis-2/pom.xml

```diff
@@ -2290,6 +2296,11 @@
                         <groupId>org.slf4j</groupId>
                         <artifactId>slf4j-api</artifactId>
                     </exclusion>
+<!--        Junit 4 cannot be excluded yet -->
```

Comment:
```suggestion
<!-- Junit 4 cannot be excluded yet, because of this issue -->
<!-- https://github.com/testcontainers/testcontainers-java/issues/970 -->
```